scripts/analysis.py

```diff
@@ -663,7 +804,51 @@ def plot_power(cursor):
     con = lite.connect(file)
     with con:
         cur = con.cursor()
-        print(snf(cur))
-        plot_power(cur)
-        plot_in_out_flux(cur, 'source', False, 'source vs time', 'source')
-        plot_in_out_flux(cur, 'sink', True, 'isotope vs time', 'sink')
+        # print(snf(cur))
+        # plot_power(cur)
+        # plot_in_out_flux(cur, 'source', False, 'source vs time', 'source')
+        # plot_in_out_flux(cur, 'sink', True, 'isotope vs time', 'sink')
+        init_year, init_month, duration, timestep = get_sim_time_duration(cur)
+        """
```

Comment:
The intent for this one was to remind myself (and the user?) to show which key reflects which metric. But I will note the # instead of docstring!